migration-guide/business-central.adoc

```diff
@@ -0,0 +1,150 @@
+It included capabilities for monitoring, administration, and management of business automation assets.  With the release of both IBM {DEV_TOOLS} and IBM {CANVAS} 
+most of the asset management capabilities were replaced or sometimes take a different approach altogether.  This section describes 'the' migration of business 
+assets 'that are' created in Business Central so that they can be maintained by developers by using either {DEV_TOOLS} or {CANVAS} or both.
+== Prerequisites
+
+The following tools are required to migrate from Business Central to {DEV_TOOLS}. It is assumed that you have a current version of Business Central running to export 
+certain types of assets to their supported form in {PRODUCT_SHORT}:
+
+include::../partials/prerequisites.adoc[]
+
+== Business Automation Assets in Businss Central
+
+The following types of business automation assets were available within Business Central with a focus on the the assets that are deprecated in Version 9.0 too. 
+This is to indicate how to manage assets in absence of Business Central.
+
+.Business Central Asset Types
+[cols="2,1,4,3,3"]
+|===
+| Asset Name | Asset Type | Description | Version 9.0 Supported Tools | Additional Info.
+
+| Business Process | Process (.bpmn) | Modeled as Business Process & Notation (BPMN) diagrams | {DEV_TOOLS} +
+{CANVAS} | Also used for straight-through-processing (STP) as well as rule-flow
+| Decision Model | Decision (.dmn) | Modeled as Decision Modeling & Notation (DMN) diagrams | {DEV_TOOLS} +
+{CANVAS} |
+| Technical Rule | Decision (.drl) | Drools Rule Language, used for technical rules | {DEV_TOOLS} |
+| Decision Table (Spreadsheet) | Decision (.xls) | DRL-based decision table, modeled as an Excel spreadsheet | {DEV_TOOLS} +
+MS Excel Online |
+| Test Scenario | Decision (.scesim) | Test scenario used for decision unit testing | {DEV_TOOLS} |
+| Data Object | Model (.java) | Java data object used by rules and processes | {DEV_TOOLS} | Define as separate Maven dependency project
+| Global Variable(s) | Decision (.drl) | Vocabulary definition | {DEV_TOOLS} | Managed directly in .drl files
+| Domain Specific Language Definition | Decision (.dsl) | Vocabulary definition | **Deprecated** |
+| Enumeration | Model | Vocabulary definition | {DEV_TOOLS} | Managed directly in .drl files
+| Form | Forms (.frm) | Human Form | **Deprecated** | New form builder in future release
+| Guided Decision Table | Decision (.gdst) | Guided Decisison Table | **Deprecated** | **Export** from BC to Decision Table Spreadsheet (XLS)
+| Guided Decision Table Graph | Decision (.gdst-set) | Guided Decision Table Graph| **Deprecated** | 
+| Guided Rule | Decision (.rdrl) | Guided Technical Rule | {DEV_TOOLS} | **Copy** source from BC to new Technical Rule (.drl) file
+| Guided Rule Template | Decision (.template) | Template for guided rules | **Deprecated** |
```

Comment:
I tend to think `.template` content can be migrated in the same way as we guide users for `.rdrl`. The only think is that `.template` produces more rules , and I am not sure if `drl` files accepts to contain more rules in single file.

Reply:
Are we okay saying this is deprecated, or do we need something more here?

Reply:
well, that is the question I can not answer probably, I am just saying that also `.template` files should be easy to migrate in same way as `.rdrl`, however yes, maybe we do not want to support, for example due to our capacities of support or something

maybe @eduardocerqueira or @porcelli could advise

Reply:
> well, that is the question I can not answer probably, I am just saying that also `.template` files should be easy to migrate in same way as `.rdrl`, however yes, maybe we do not want to support, for example due to our capacities of support or something
> 
> maybe @eduardocerqueira or @porcelli could advise

agree, changed .templates just like .rdrl